docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,67 @@
+Pulse integrates with Jira Cloud to receive:
+
+-   Data about **incidents**, necessary to calculate the following Accelerate metrics:
+
+    -   [Time to recover](../metrics/accelerate.md#time-to-recover)
+
+    -   [Change failure rate](../metrics/accelerate.md#change-failure-rate)
+
+-   Data bout **issues**, necessary to calculate [Lead time and Cycle time](../metrics/lead-cycle-time.md).
 
 ## Setting up the Jira integration
 
+!!! note
+    -   The Jira integration must be set up by a **Jira administrator**, as Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    -   You can only integrate one Pulse organization with each Jira instance.
+    -   [See below](#jira-permissions) the permissions that Pulse requires from your Jira instance.
+
 To set up the Jira integration:
 
 1.  On Pulse, [expand **Integrations** and select **Jira**](https://app.pulse.codacy.com/integrations/jira){: target="_blank"}.
 
-1.  Click **Install Jira** and follow the instructions on the Jira UI to install the app.
+1.  Click **Install Jira App** and follow the instructions on the Jira UI to install the app. You must be a Jira administrator to proceed with this step.
 
-    !!! important
-        This step should be performed by a Jira administrator, since Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    ![Installing the Pulse Jira app](images/jira-installing.png)
 
-        You can only integrate one Pulse organization with each Jira instance.
+1.  Confirm that Pulse successfully created the integration and the necessary webhooks on Jira. If there was an error please [contact support](mailto:pulsesupport@codacy.com).
 
-    ![Installing the Pulse Jira app](images/jira-installing.png)
+<!--TODO Add screenshot for successful installation-->
+
+1.  Choose the strategy to detect incidents that best fits your workflows. See the [section below](#incident-detection-strategy) for a detailed description of each option.
+
+<!--TODO Add screenshot for incident strategy-->
+
+1.  Click **Complete setup**.
+
+Your Jira integration is now complete. Pulse will start loading your data for the last 90 days.
 
-1.  Wait until you get a confirmation that Pulse successfully created the integration and the necessary webhooks on Jira.
+<!--TODO Add screenshot for setup ok-->
 
-    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The Pulse Jira integration can detect **incidents** automatically using [issues assigned with the label "Incident"](#jira-incident-label).
+
+You can also choose [not to detect incidents via Jira](#jira-incident-not-detect).
+
+### Use issues assigned with the label "Incident" {: id="jira-incident-label"}
+
+-   <!--TODO Add detection details-->
```

Comment:
An issue is considered an incident if:
- It's in a terminal status (aka it's closed). In concrete Jira terms, a status of category 3 'Done' -- https://support.atlassian.com/jira-work-management/docs/workflows-and-statuses-for-the-board/
- It has the label "Incident", matched in a case insensitve manner.

If this is true, then for each value in the 'Components' field Pulse will create an incident event for the corresponding system, so values in the 'Components' field should correspond to Pulse systems (case-sensitive match).

- If a component does not already exist in Pulse as a system, no incident event will be created for it.
- If ALL components do not already exist in Pulse as systems, an incident will be created and attributed to the catch-all system `_unknown_`.

- If an issue has the "Incident" label, closing it will create an incident (or multiple, depending on the components field).
- If an issue is closed, adding the "Incident" label will create an incident (or multiple, depending on the components field).
- If an issue is closed and has the "Incident" label, adding and/or removing components will create and/or delete the corresponding incidents.
- If an issue is closed and has the "Incident" label, re-opening the issue, removing the "Incident" label or deleting the issue will delete the corresponding incidents.

Reply:
I updated this section. Please validate if it's ok.